### ISSUE TITLE
[Bug]: Error message on uploading Logo shows 100MB instead of 1MB 

### ISSUE THREAD

**Issue by chandannkumar** [appsmithorg/appsmith#23168]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Since uploading logo limit size is 1MB, but on it throws error message saying `Payload too large: File size cannot exceed 100MB` instead of 1MB as uploading file more than 1mb is not allowed

![Screenshot (278)](https://github.com/appsmithorg/appsmith/assets/95341241/0137df8a-35be-473b-b914-affa8ae687b4)

### Steps To Reproduce

1. create a application with multiple page
2. Go to App Settings> navigation> Upload Logo
3. Try to upload valid image larger than 1MB
4. Observe the error says 100MB instead of 1MB

### Environment

Deploy Preview

### Version

Cloud

**Comment by harshitpandey0426** [comment]
This message comes from messages.ts file. Which is
> Payload too large. File size cannot exceed ${maxFileSize}MB.

and maxFileSize comes from ApiUtils.ts. Which is
> createMessage(ERROR_413, 100)
#### Changed this maxFileSize from 100 to 1

I have raised a PR for this bug, please have a look. @somangshu @dhruvikn

**Comment by harshitpandey0426** [comment]
@somangshu @dhruvikn How to get maintainers approval to run tests on my PR?

**Comment by somangshu** [comment]
Thanks for raising the PR @harshitpandey0426, Ill take it forward from here. Ill keep you posted on the PR is anything is needed!

**Comment by dhruvikn** [comment]
Hey @harshitpandey0426, thanks for taking a look at the issue. I [saw your PR](https://github.com/appsmithorg/appsmith/pull/23310) and disagree with the changes you have made. We don't want to change the `apiFailureResponseInterceptor`'s limit to 1 MB. This function is called with all APIs and we have deliberately set the limit to 100 MB to avoid any large file uploads. 

Coming back to the issue at hand, we need to triage why this is happening. It could be due to anything such as - 
1. The frontend validation of the logo upload failed and allowed a file size bigger than 1 MB.
2. The backend validation failed for the same reason.
3. Or anything else. 

If you'd still like to contribute, I would request you to triage the issue first and post your findings here. Based on that, we can come up with a plan and include all the relevant people (backend and/or product folks) in our conversation.

For now, I'm requesting changes to your PR to avoid merging. 

cc: @somangshu, do you think we should close the PR instead?

**Comment by harshitpandey0426** [comment]
Hey @dhruvikn,
Thanks for pointing this out. I will have another look at it and share if I find anything relevant

**Comment by harshitpandey0426** [comment]
For some reason I dont see Admin Settings in local setup. Any suggestion how can I enable this?

**Comment by dhruvikn** [comment]
You will need a [business edition license](https://www.appsmith.com/pricing) for that.

May I ask why you need admin settings in your local setup and how are they related to this issue?

**Comment by harshitpandey0426** [comment]
I was going through other flows to add logo and check validation. One other flow which I found was admin Settings -> Branding -> logo.

**Comment by dhruvikn** [comment]
The issue is still reproducible in release. I have triaged this issue. My findings are -
1. The FE validation is failing. It was set to 2 MB on the FE when BE accepts a max size of 1 MB.
2. The BE validation is also failing to an extent. It should have thrown the error that the file size cannot exceed 1 MB, but instead, it threw the error mentioned in the issue's description. @NilanshBansal can you please check once? I'm not sure but I think it could be related to Line 97-107 of  `appsmith/app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/AssetServiceCEImpl.java`

<img width="781" alt="CleanShot 2023-06-02 at 01 22 53@2x" src="https://github.com/appsmithorg/appsmith/assets/22471214/6692f627-4d90-4522-bed1-dea7a4d672ab">

@harshitpandey0426, do you want to pick the FE change up? It should be a teeny-tiny PR.

**Comment by dhruvikn** [comment]
Also, marking this as low priority since the feature isn't [available to public at the moment](https://github.com/appsmithorg/appsmith/pull/23596).

**Comment by harshitpandey0426** [comment]
@dhruvikn Sure I will check this

**Comment by harshitpandey0426** [comment]
@dhruvikn In release I dont see "show logo" toggle. I can see this toggle in production though
![image](https://github.com/appsmithorg/appsmith/assets/54776298/8aae0e1f-8b7b-4e32-b928-bea5a2368a5b)

**Comment by dhruvikn** [comment]
That's because the feature is hidden behind a feature flag. If you want to test this, you'll need to setup local backend server and change the feature flag rule in the backend. 

However, since it's a very small change in the validation file, I'd suggest you make the change, raise a PR, and our QA folks will test it on your behalf. Let me know if this works.

**Comment by harshitpandey0426** [comment]
@dhruvikn Yes I have the local backend server running. If possible let me know what changes I have to make in backend

**Comment by harshitpandey0426** [comment]
@dhruvikn I was able to test. I have opened a PR.

**Comment by dhruvikn** [comment]
@NilanshBansal, can we please open another issue to handle this on backend as well? Please check my [thoughts here](https://github.com/appsmithorg/appsmith/issues/23168#issuecomment-1572684688) and take a call/assign anyone else as needed.

**Comment by NilanshBansal** [comment]
> The BE validation is also failing to an extent. It should have thrown the error that the file size cannot exceed 1 MB, but instead, it threw the error mentioned in the issue's description. @NilanshBansal can you please check once? I'm not sure but I think it could be related to Line 97-107 of appsmith/app/server/appsmith-server/src/main/java/com/appsmith/server/services/ce/AssetServiceCEImpl.java

@dhruvikn the above error is not coming from backend. The backend response is as shown below.
<img width="1099" alt="SCR-20230608-ofg" src="https://github.com/appsmithorg/appsmith/assets/25542733/7d7c734a-e340-4666-b1c1-1c1b70fe0088">
